lib/vm/src/compatability.rs

```diff
@@ -102,11 +104,11 @@ mod test {
         let symbols = wasm_nm::symbols(PUBLIC_SYMBOLS.clone(), &mut reader).unwrap();
 
         // if contract has more than we provide, bad
-        let imports_good = import_requirements_satisfied(&symbols, &["c_read", "c_write"]);
-        assert_eq!(imports_good, false);
+        let imports_good = find_missing_import(&symbols, &["c_read", "c_write"]);
```

Comment:
Didn't you change these to `db_read` and `db_write`?
Or was that PR not merged yet?

The logic looks good, just surprised by the naming.

Reply:
Those are test vectors that are compatible to `let mut reader = std::io::Cursor::new(CONTRACT_0_6);`. Thus the old names. Fortunately, `find_missing_import` is agnostic of the import names, which is demostrated by this test.